src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java

```diff
@@ -28,32 +28,30 @@
  * @author Josue Rodriguez, Alejandro Sánchez Luizaga
  * @version 1.0
  */
-public class ConvertPPTtoImage implements IConvert{
+public class ConvertPptxToImage implements IConvert{
     /**
      * Implements convert(CriteriaConvert) from IConvert interface.
      * Conversion is performed via Apache poi utility.
      *
      * @param criteriaConvert holds source and destination file paths and output file extension.
      */
     public void convert(CriteriaConvert criteriaConvert) {
-        CriteriaPPTtoImage criterion = (CriteriaPPTtoImage) criteriaConvert;
+        CriteriaPptxToImage criteria = (CriteriaPptxToImage) criteriaConvert;
         try {
-            File file = new File(criterion.getSrcPath());
+            File file = new File(criteria.getSrcPath());
             XMLSlideShow ppt = new XMLSlideShow(new FileInputStream(file));
             Dimension pgsize = ppt.getPageSize();
             List<XSLFSlide> slide = ppt.getSlides();
             BufferedImage img = null;
             for (int i = 0; i < slide.size(); i++) {
                 img = new BufferedImage(pgsize.width, pgsize.height, BufferedImage.TYPE_INT_RGB);
                 Graphics2D graphics = img.createGraphics();
-
                 graphics.setPaint(Color.white);
                 graphics.fill(new Rectangle2D.Float(0, 0, pgsize.width, pgsize.height));
```

Comment:
That is "(0, 0, pgsize.width, pgsize.height)" 0 look magical numbers.

Reply:
fix, complete